### PR TITLE
#569 Drop and recreation of dataplane_msms table and view_dataplane_msms v…

### LINKED ISSUE #569: Drop and recreation of dataplane_msms table and view_dataplane_msms v…
This issue introduces changes in "dataplane_msms" table and "view_dataplane_msms" view, in order to support the new design of the "dataplane_view" module.

### PR DESCRIPTION
**Pull request by georgeepta**
### Description of PR
This pull request introduces changes in "dataplane_msms" table and "view_dataplane_msms" view, in order to support the new design of the "dataplane_view" module.

What component(s) does this PR affect?

- [ X ] Back-End (Database, Detection/Configuration/etc. Microservices)
- [ ] Front-End (Flask, API, UI, etc)
- [ ] Monitor (RIPE RIS, BGPStream RV/RIS/CAIDA, etc.)
- [ ] Docs
- [ ] Build System

Does the PR require changes on other components? If yes, please mark the components:

- [ X ] Back-End (Database, Detection/Configuration/etc. Microservices)
- [ ] Front-End (Flask, API, UI, etc)
- [ ] Monitor (RIPE RIS, BGPStream RV/RIS/CAIDA, etc.)
- [ ] Docs
- [ ] Build System

### Related Issue

Resolves #569

### Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ X ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Docs update
- [x] None of the above

### Checklist:

- [ X ] I have read the **[contributing guide](https://github.com/FORTH-ICS-INSPIRE/artemis/blob/master/CONTRIBUTING.md)** and my code conforms to the guidelines.
- [ ] This change requires a change in the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by vkotronis** [comment]
@georgeepta I will review and merge this within the week. Thank you for the PR!

File: other/db/data/tables.sql [hunk 1]
```diff
@@ -241,18 +241,20 @@ CREATE FUNCTION search_bgpupdates_by_as_path_and_hijack_key(key text, as_paths B
 $$ LANGUAGE sql STABLE;
 
 CREATE TABLE IF NOT EXISTS dataplane_msms (
-    key VARCHAR ( 32 ) NOT NULL,
+    key VARCHAR (32) NOT NULL,
```
Comment (vkotronis) [inline other/db/data/tables.sql:244]: please also apply the following changes w.r.t. msms table and view on `testing/detection/db/data/tables.sql`

**Review by vkotronis — CHANGES_REQUESTED** [review]
@georgeepta Seems good at first sight, just apply the changes you made also in the testing `tables.sql`

File: other/db/data/tables.sql (hunk 1 above)
Comment (vkotronis) [reply other/db/data/tables.sql:244]: btw do not worry about any travis-ci errors they relate to this being a forked PR, I will merge the PR securely after you also apply the change in the test tables.sql

Comment (georgeepta) [reply other/db/data/tables.sql:244]: @vkotronis Οκ! I fixed it.

**Review by vkotronis — APPROVED** [review]

**Pull request merged**